lib/dor/services/client/release_tags.rb

```diff
@@ -30,12 +30,14 @@ def create(tag:)
 
         # List release tags for an object
         #
+        # @param public [Boolean] indicates if we only want public release tags (defaults to false)
         # @raise [NotFoundResponse] when the response is a 404 (object not found)
         # @raise [UnexpectedResponse] if the request is unsuccessful.
         # @return [Array<ReleaseTag>]
-        def list
+        def list(public: false)
+          params = public ? 'public=true' : ''
           resp = connection.get do |req|
-            req.url "#{api_version}/objects/#{object_identifier}/release_tags"
+            req.url "#{api_version}/objects/#{object_identifier}/release_tags?#{params}"
```

Comment:
allows us to support this existing param in DSA that we can use in common-accessioning to only get the latest tags: https://github.com/sul-dlss/dor-services-app/blob/main/app/controllers/release_tags_controller.rb#L18